recipes/nnlojet/recipe.yaml

```
@@ -0,0 +1,85 @@
+context:
+  name: "nnlojet"
+  version: "1.0.0"
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://nnlojet.hepforge.org/nnlojet-v${{ version }}.tar.gz
+  sha256: b02013922387cf37745487d23b271eaae257c8af76bb3cca6bbf885dd8139e74
+
+build:
+  number: 0
+  script:
+    # Get an updated config.sub and config.guess
+    - cmake "${CMAKE_ARGS}" -DCMAKE_INSTALL_PREFIX="${PREFIX}" -S . -B build
+    # - if: win
+    #   then: cmake %CMAKE_ARGS% -S . -B build
+    - cmake -LH build
+    - cmake --build build --parallel="${CPU_COUNT}"
+    # - if: win
+    #   then: cmake --build build --config=Release --parallel="%CPU_COUNT%"
+    - cmake --install build
+
+requirements:
+  run_exports:
+    - ${{ pin_subpackage('nnlojet', upper_bound='x.x') }}
+  build:
+    - ${{ stdlib('c') }}
+    - ${{ compiler('cxx') }}
+    - ${{ compiler('fortran') }}
+    - lhapdf
+    - python
```

```suggestion
    - if: build_platform != target_platform
      then:
        - python
        - cross-python_${{ target_platform }}
```

I had commented that `python` should be a `host` requirement, but for (future) cross-platform builds it will also need to be a [`build` requirement](https://rattler.build/latest/reference/recipe_file/#build).